### PR TITLE
employee package and class commited by sharun

### PR DESCRIPTION
**Pull request by ghost**
Hi yoganand,

--added new package

--added new employee class

Thanks 
Sharun

### CODE REVIEW

File: src/test/java/employee/employee.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package employee;
+
+public class employee {
+
+	public void login()
```
Comment (yoganandbheema23) [inline src/test/java/employee/employee.java:7]: Hi Sharun,
Where is the @Test annotation in this login method.
Thanks ,
YOganand

**Review by yoganandbheema23 — CHANGES_REQUESTED** [review]
please add @ Test annotation

File: src/test/java/employee/employee.java (hunk 1 above)
Comment (ghost) [reply src/test/java/employee/employee.java:7]: Coded added by , please review and commit the changes.
Thanks,
Sharun

Comment (yoganandbheema23) [reply src/test/java/employee/employee.java:7]: done

**Pull request merged**